examples/Multi_Lingual_Customer_Support_Response_Pipeline.ipynb

```diff
@@ -0,0 +1,222 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "FLGnrbqamw30"
+      },
+      "source": [
+        "Requirements:\n",
+        "1. Instill Core and the Instill SDK.\n",
+        "2. Models from Hugging Face or OpenAI for language detection, sentiment analysis, and response generation.\n",
+        "3. Language libraries, such as googletrans for fallback translation."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "CLCDA-tLnatt"
+      },
+      "source": [
+        "1. Install required libraries:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Kt_yutMVnW7z"
+      },
+      "outputs": [],
+      "source": [
+        "pip install instill-sdk googletrans==4.0.0-rc1 transformers torch\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Zuvuqp-angT6"
+      },
+      "source": [
+        "2. Set Up Instill SDK and Initialize API:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "4eRArD3TnjOj"
+      },
+      "outputs": [],
+      "source": [
+        "from instill_sdk import InstillCoreClient\n",
```

Comment:
```suggestion
        "from instill_sdk import InstillCoreClient\n",
```
This is not how to use our SDK. Please refer to our [documentation](https://www.instill.tech/docs/sdk/python).

```Python
from instill.clients import init_core_client

client = init_core_client(api_token="instill_sk_***")
```